go/test/endtoend/utils/utils.go

```diff
@@ -232,6 +236,47 @@ func WaitForAuthoritative(t *testing.T, vtgateProcess cluster.VtgateProcess, ks,
 	}
 }
 
+// WaitForColumn waits for a table's column to be present
+func WaitForColumn(t *testing.T, vtgateProcess cluster.VtgateProcess, ks, tbl, col string) error {
```

Comment:
nit: since you are sending in `t *testing.T`, we don't really need to return an error, right? we can just fail in here, right?

Reply:
I just want the author of the test to decide if they want to continue or fail.